src/themes.js

```diff
@@ -6,6 +6,11 @@ export const light = {
     minutes: '#ccc',
     hour: '#000',
     tick: '#000',
+    smallTickWidth: 2,
+    largeTickWidth: 4,
+    secondHandWidth: 3,
+    minuteHandWidth: 6,
+    hourHandWidth: 8,
```

Comment:
These aren't needed, right? Since they are the defaults? Same for all the theme updates in this file.

Reply:
I added these here more for documentation's sake since there hasn't been an update to the documentation surrounding how to create custom themes. They definitely can be removed. I can make this change is required.